velox/connectors/hive/iceberg/PositionalDeleteFileReader.cpp

```diff
@@ -223,6 +223,16 @@ void PositionalDeleteFileReader::updateDeleteBitmap(
       deletePositionsVector->as<FlatVector<int64_t>>()->rawValues();
   int64_t offset = baseReadOffset + splitOffset_;
 
+  // If the offset is greater than the last position in this delete rows batch,
+  // nothing to delete.
+  if (offset > deletePositions[deletePositionsVector->size() - 1]) {
+    return;
+  }
+
```

Comment:
Can you please add a comment here:
// Skip the delete positions in deletePositionsVector until they are in the [offset, rowNumberUpperBound) range.

Reply:
In a seperate commit, maybe rename rowNumberUpperBound to endOffset and offset to beginOffset